.github/workflows/generate-package-metadata.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - id: set-matrix
-        run: echo "::set-output name=matrix::$(python generate_package_list.py)'"
+        run: echo "matrix=$(python generate_package_list.py)'" >> $GITHUB_OUTPUT
```

Comment:
there is a trailing `'` here. This was present before. Was this a typo or something that always existed or is it some type of delimiter or something?

Reply:
I'd reckon the `'` is a typo and should be removed. This is fed into a JSON parse so it'll be quite obvious if it fails.

Reply:
thank you @danielrbradley!